vita3k/crypto/include/crypto/aes.h

```diff
@@ -164,13 +163,13 @@ int aes_crypt_cfb128( aes_context *ctx,
  *
  * \return         0 if successful
  */
-int aes_crypt_ctr( aes_context *ctx,
-                       size_t length,
-                       size_t *nc_off,
-                       unsigned char nonce_counter[16],
-                       unsigned char stream_block[16],
-                       const unsigned char *input,
-                       unsigned char *output );
+int aes_crypt_ctr(aes_context *ctx,
+    size_t length,
+    size_t *nc_off,
+    unsigned char nonce_counter[16],
+    unsigned char stream_block[16],
+    const unsigned char *input,
+    unsigned char *output);
 
 void aes_cmac(aes_context *ctx, int length, unsigned char *input, unsigned char *output);
```

Comment:
I probably wouldn't refactor the aes files... they aren't from out project.

Reply:
i have no refactor, is just clang format not runed for this file before, pup pr have runned clang formart no in last commit, so you can see lot file no have get clang format